app/pub_sub/subscribers/site_activity.rb

```diff
@@ -0,0 +1,33 @@
+module Subscribers
+  class SiteActivity < ::Subscribers::Base
+    def site_created(event)
+      return if event.payload[:subject].nil? || event.payload[:subject].new_record?
```

Comment:
I totally agree, I wasn´t sure of this approach, but once implemented it doesn´t makes sense to populate the system with messages that need to be ignored!